source/adapters/opencl/kernel.cpp

```diff
@@ -408,3 +408,28 @@ UR_APIEXPORT ur_result_t UR_APICALL urKernelSetArgSampler(
   CL_RETURN_ON_FAILURE(RetErr);
   return UR_RESULT_SUCCESS;
 }
+
+UR_APIEXPORT ur_result_t UR_APICALL urGetKernelSuggestedLocalWorkSize(
+    ur_queue_handle_t hQueue, ur_kernel_handle_t hKernel, uint32_t workDim,
+    const size_t *pGlobalWorkOffset, const size_t *pGlobalWorkSize,
+    size_t *pSuggestedLocalWorkSize) {
+  std::ignore = hQueue;
+  std::ignore = hKernel;
+  std::ignore = workDim;
+  std::ignore = pGlobalWorkOffset;
+  std::ignore = pGlobalWorkSize;
+
+  pSuggestedLocalWorkSize[0] = 1;
+  if (workDim >= 2)
+    pSuggestedLocalWorkSize[1] = 1;
+  if (workDim >= 3)
+    pSuggestedLocalWorkSize[2] = 1;
+
+  // cl_int RetErr = clGetKernelSuggestedLocalWorkSizeKHR(
```

Comment:
remove unused comment.

Reply:
I add some comments here. I will be used in the future.